admin_guide/high_availability.adoc

```diff
@@ -10,50 +10,212 @@
+    $ORIGIN v3.rhcloud.com.
+
+    @       IN      SOA     . v3.rhcloud.com. (
+                         2009092001         ; Serial
+                             604800         ; Refresh
+                              86400         ; Retry
+                            1206900         ; Expire
+                                300 )       ; Negative Cache TTL
+            IN      NS      ns1.v3.rhcloud.com.
+    ns1     IN      A       127.0.0.1
+    *       IN      A       10.245.2.2
+            IN      A       10.245.2.3
+
+
+----
+====
+
+. Test the entry. The following example test uses `dig` (available in the
+*bind-utils* package) in a *Vagrant* environment to show multiple answers for
+the same lookup. Performing multiple pings shows the resolution swapping between
+IP addresses:
++
+[options="nowrap"]
+====
+
+----
+
+$ dig hello-openshift.shard1.v3.rhcloud.com
+
+; <<>> DiG 9.9.4-P2-RedHat-9.9.4-16.P2.fc20 <<>> hello-openshift.shard1.v3.rhcloud.com
+;; global options: +cmd
+;; Got answer:
+;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 36389
+;; flags: qr aa rd; QUERY: 1, ANSWER: 2, AUTHORITY: 1, ADDITIONAL: 2
+;; WARNING: recursion requested but not available
+
+;; OPT PSEUDOSECTION:
+; EDNS: version: 0, flags:; udp: 4096
+;; QUESTION SECTION:
+;hello-openshift.shard1.v3.rhcloud.com. IN A
+
+;; ANSWER SECTION:
+hello-openshift.shard1.v3.rhcloud.com. 300 IN A	10.245.2.2
+hello-openshift.shard1.v3.rhcloud.com. 300 IN A	10.245.2.3
+
+;; AUTHORITY SECTION:
+v3.rhcloud.com.		300	IN	NS	ns1.v3.rhcloud.com.
+
+;; ADDITIONAL SECTION:
+ns1.v3.rhcloud.com.	300	IN	A	127.0.0.1
+
+;; Query time: 5 msec
+;; SERVER: 10.245.2.3#53(10.245.2.3)
+;; WHEN: Wed Nov 19 19:01:32 UTC 2014
+;; MSG SIZE  rcvd: 132
+
+$ ping hello-openshift.shard1.v3.rhcloud.com
+PING hello-openshift.shard1.v3.rhcloud.com (10.245.2.3) 56(84) bytes of data.
+...
+^C
+--- hello-openshift.shard1.v3.rhcloud.com ping statistics ---
+2 packets transmitted, 2 received, 0% packet loss, time 1000ms
+rtt min/avg/max/mdev = 0.272/0.573/0.874/0.301 ms
+
+$ ping hello-openshift.shard1.v3.rhcloud.com
+[...]
+----
+
+====
+////
+
+== Configuring a Highly-available Routing Service
+The following steps describe how to setup a highly available router environment with IP failover:
+
+. Label the nodes for the service. This step can be optional if you run the
+service on any of the nodes in your Kubernetes cluster and use VIPs that can
+float within those nodes. This process may already exist within a complex
+cluster, in that nodes may be filtered by any constraints or requirements
+specified (e.g. nodes with SSD drives, or higher cpu/memory/disk requirements,
+etc.).
++
+The following example defines a label as router instances that are servicing
+traffic in the US west geography "ha-router=geo-us-west":
++
+----
+$ osc label nodes openshift-minion-{5,6,7,8,9} "ha-router=geo-us-west"
+----
+
+. Run the IP service (router) with two or more replicas As described earlier,
```

Comment:
Looks like there's 2 sentences here. So ```Run the IP service (router) with two or more replicas.``` maybe? Missing the period/dot at the end of the first sentence.